### DISCUSSION TITLE
External temperature

### DISCUSSION THREAD

**Discussion by marnixt** [DomiStyle/esphome-panasonic-ac#74] [Q&A]
I'm currently suffering from some erratic behavior of my Panasonic Airco's due to the fact that it measures the temperature in the unit. It tends to become very hot since it's close to the ceiling causing the unit to go idle before the desired room temperature is reached. 

Does the external temperature feature mean I can feed any temperature from Home Assistant to the Panasonic and it would then use that instead of the internal temperature sensor?
That would be an absolute killer feature for me since it would then basically allow me to fine-tune the airco behavior to my needs.

**Comment by DomiStyle** [comment]
No, the external temperature sensor is what's sent to Home Assistant if your unit does not report the temperature on its own.

There's some discussion going on in [this issue](https://github.com/DomiStyle/esphome-panasonic-ac/issues/50) about using it to essentially set the state of the AC automatically but it is currently not implemented.

**Reply by marnixt** [reply]
Thanks, the discussion in that referenced issue is what I'm looking for. Will continue there
